### DISCUSSION TITLE
Animation toggle based on power mode.

### DISCUSSION THREAD

**Discussion by connorpink** [mrkai77/Loop#424] [Q&A]
I like this app and I like the animation of the window movement.
 My idea is to have the advanced setting that animates the window resize have an option to auto toggle with respect to the power mode.
My reasoning for this feature is that on my MacBook pro when it is in low power mode the screen is at 60hz and I wouldn't want to have the animation on. However when its not on low power mode the screen is 120hz and I think the animations look much better.

I would like the animation to toggle on when not on low power mode and toggle back off when on low power mode.

This could be a sub-setting under "Animate Window resize" that will disable this setting when toggled on, and be titled something like "toggle animate rezise with power mode"

**Comment by mrkai77** [comment]
Moved to #444
